src/webhook/user_validation.py

```diff
@@ -0,0 +1,72 @@
+from flask import request, Blueprint
+import sys, traceback
+import os
+from prometheus_client import Counter
+from webhook.request_helper import validate, responses
+
+bp = Blueprint("user-webhook", __name__)
+
+# define what we track, declare Counter, how many times this route is accessed
+TOTAL_USER = Counter('webhook_user_validation_total', 'The total number of user validation requests')
+DENIED_USER = Counter('webhook_user_validation_denied', 'The total number of user validation requests denied')
+
+user_suffix = "@redhat.com"
```

Comment:
Recommend renaming `user_suffix` to something more descriptive like `protected_user_suffix`